packages/inferno-mobx/src/Provider.ts

```diff
@@ -51,19 +51,18 @@ export default class Provider extends Component<any, any> {
 
 if (process.env.NODE_ENV !== 'production') {
 	Provider.prototype.componentWillReceiveProps = function(nextProps) {
-
 		// Maybe this warning is to aggressive?
 		if (Object.keys(nextProps).length !== Object.keys(this.props).length) {
 			warning(
 				'MobX Provider: The set of provided stores has changed. ' +
-				'Please avoid changing stores as the change might not propagate to all children'
+					'Please avoid changing stores as the change might not propagate to all children'
```

Comment:
This is intentional, it's to highlight that it's the continuation of the previous string, and not a new argument.